src/components/app_catalog/app_detail/InstallAppModal.js

```diff
@@ -124,6 +127,24 @@ const InstallAppModal = props => {
     reader.readAsText(files[0]);
   };
 
+  const updateSecretsYAML = files => {
+    var reader = new FileReader();
+
+    reader.onload = (function() {
+      return function(e) {
+        try {
+          let parsedYAML = yaml.safeLoad(e.target.result);
+          setSecretsYAML(parsedYAML);
+          setSecretsYAMLError('');
+        } catch (err) {
```

Comment:
Also, [you can exclude the parameter, since it's not used](https://v8.dev/features/optional-catch-binding)